src/test/test_statement.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+Unit test for the p2p statement class
+
+Copyright 2021-12-12 AlexanderLill
+"""
+import unittest
+
+from Statement import Statement
```

Comment:
I stumbled upon an import error in my local environment for some reason. Could you please update it to?

```suggestion
from src.Statement import Statement
```

Thank you.